changelogs/fragments/67-cache.yaml

```diff
@@ -0,0 +1,5 @@
+---
+bugfixes:
+  - Remove improper usage of ``@functools.cache`` on async functions in the
+    ``antsibull_core.ansible_core`` module
+    (https://github.com/ansible-community/antsibull-core/pull/67).
```

Comment:
I forget, what's the proper way to handle changelogs when backporting? `@functools.cache` should be changed to `@functools.lru_cache`, but I'm not sure about the URL.

Reply:
Usually we don't modify the fragments; in this case, I would probably change the content, and add the new URL (so both are there).

Reply:
Right, I thought that was the general rule, but I wasn't sure what to do about this one.

Reply:
https://github.com/ansible-community/antsibull-core/pull/69/commits/00551a1e60c64406e713071d47b286b953e04fd9